browser-extension/src/addTournesolRecommendations.js

```diff
@@ -134,21 +153,34 @@ const getLocalizedActionError = () => {
 const createBanner = () => {
   const banner = document.createElement('div');
   banner.id = 'tournesol_banner';
+  banner.className = 'tournesol_banner';
+
+  // Only display the banner if the user didn't explicitly close it.
+  chrome.storage.local.get('displayBanner', (items) => {
```

Comment:
If we plan to reuse this mechanism for other banners, a more specific key should be used.
Something like `"displayBannerStudy2023"` ?

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1220/commits/8f5fd25d2a2bc6dbcce3a6b7a8eb6c3461251e53